alembic/versions/231a9abfb3b8_adding_deposit_addr_detail.py

```diff
@@ -1,30 +0,0 @@
-"""adding_deposit_addr_detail
```

Comment:
Are the deletions expected?

Reply:
Yes sir, i just combined all alembic versions into a single file